discovery-provider/src/tasks/tracks.py

```diff
@@ -277,7 +277,6 @@ def refresh_track_owner_ipfs_conn(owner_id, session, update_task):
         session.query(User.creator_node_endpoint)
         .filter(
             User.is_current == True,
-            User.is_ready == True,
```

Comment:
this change is bc i noticed that our refresh_track_owner_ipfs_conn() method only works on ready users, which is an unnecessary restriction